sdk/include/reactos/mc/net_msg.mc

```diff
@@ -2303,35 +2304,47 @@ devicename         Is one or more printers (LPT1: through LPT9:)
 NET HELP command | MORE displays Help one screen at a time.
 .
 Language=Polish
-NET SHARE makes a server's resources available to network users. When
-used without options, it lists information about all resources being
-shared on the computer. For each resource, Windows reports the
-devicename(s) or pathname(s) and a descriptive comment associated with it.
-
-sharename          Is the network name of the shared resource. Type
-                   NET SHARE with a sharename only to display information
-                   about that share.
-drive:path         Specifies the absolute path of the directory to
-                   be shared.
-/GRANT:user,perm   Creates the share with a security descriptor that gives
-                   the requested permissions to the specified user. This
-                   option may be used more than once to give share permissions
-                   to multiple users.
-/USERS:number      Sets the maximum number of users who can
-                   simultaneously access the shared resource.
-/UNLIMITED         Specifies an unlimited number of users can
-                   simultaneously access the shared resource.
-/REMARK:"text"     Adds a descriptive comment about the resource.
-                   Enclose the text in quotation marks.
-devicename         Is one or more printers (LPT1: through LPT9:)
-                   shared by sharename.
-/DELETE            Stops sharing the resource.
-/CACHE:Manual      Enables manual client caching of programs and documents
-                   from this share.
-/CACHE:Documents   Enables automatic caching of documents from this share.
-/CACHE:Programs    Enables automatic caching of documents and programs
-                   from this share.
-/CACHE:None        Disables caching from this share.
+NET SHARE udostępnia zasoby serwera użytkownikom sieci. Użyte
+bez opcji wyświetla informacje o wszystkich zasobach udostępnionych
+na komputerze. Dla każdego zasobu, system ReactOS zgłasza
+nazwaurzadzenia lub nazwasciezki i powiązany z nim komentarz opisowy.
+
```

Comment:
What the following part looks like in CMD? Have you checked that?

Reply:
yes, copied from cmd 2k3. run cmd and type example: net help share